docker-compose.yml

```diff
@@ -1,21 +1,31 @@
-db:
-  image: postgres
-  environment:
-    POSTGRES_USER: postgres
-    POSTGRES_PASSWORD: postgres
-cache:
-  image: library/memcached
-web:
-  image: kinto/kinto-server
-  links:
-   - db
-   - cache
-  ports:
-   - "8888:8888"
-  environment:
-    KINTO_CACHE_BACKEND: kinto.core.cache.memcached
-    KINTO_CACHE_HOSTS: cache:11211 cache:11212
-    KINTO_STORAGE_BACKEND: kinto.core.storage.postgresql
-    KINTO_STORAGE_URL: postgresql://postgres:postgres@db/postgres
-    KINTO_PERMISSION_BACKEND: kinto.core.permission.postgresql
-    KINTO_PERMISSION_URL: postgresql://postgres:postgres@db/postgres
+version: '3'
+
+services:
+  db:
+    image: postgres
+    env_file:
+      - './.env/db'
```

Comment:
I don't necessarily find it more readable to have env files. It may be completely valid on big projects, but here it's pretty straightforward don't you think? 

I'm not a super fan of having a level of indirection here, plus another folder on the repo :) But I may be wrong :) 

@Natim @glasserc what do you think?

Reply:
Maybe you right, if you underestimate yourself. It's clear, this project rocks and goes forward so that the future can justify it and look back to this checkpoint!
I think you are developing; so, what is the problem with another folder when it's **grown-up** days and days.
Think about it, you wanna create multiple instances at the same time and you have to change lots of different environment variables; it's not obvious that you _modify three lines_ of code in which define `.env` files _instead of altering all environment variables_? It's messy, don't you think? :D

Reply:
I concur with @leplatrem especially since our docker-compose file purpose is to help dev environments.

Reply:
I'm not insisting on merging all lines, especially, those in `docker-compose` which make hitches. It was an offer to increase modularity that clearly helps the environment to be more flexible. For instance, you _don't have to set a password_ and authorize all users to access the database because it's the main purpose of _dockerization_ in the realm of test and `dev-mode` as a straightforward approach; aren't you fan of simplicity in `dev-mode`? Besides, you can have _multiple users_ with _different roles_**`*`**. Don't you think one specific `docker-compose` associated with several (fully modular) `.env` files can be more **user-friendly** instead of _multiple bloody lines of ENV...ENV_ in `docker-compose` that prunes to make a headache?!
Since you're _in-depth_ and know all about angles, you might prefer a sort of simplicity that you mentioned, I respect it, meanwhile, I suggest look at it one more time _out of the box_ and decide whatever is better to interact with the community and _increasing population_ of your users.

**`*`**_**PS**_: Cleary, you can handle all roles while the access of each user is gonna be grunted via web protocols, what if some different roles can physically access the console? Thanks to docker, here's the purpose of `.env` files to verify your **security level** safe and sound.